src/data/examples/hi/10_Interaction/10_Tickle.js

```diff
@@ -1,14 +1,14 @@
 /*
- * @name Tickle
- * @description The word "tickle" jitters when the cursor hovers over.
- * Sometimes, it can be tickled off the screen.
- */
+  * @name गुदगुदी
```

Comment:
Rename it to something more meaningful please !

Reply:
ok sir